packages/wonder-blocks-data/package.json

```diff
@@ -24,4 +24,4 @@
   },
   "author": "",
   "license": "MIT"
-}
+}
```

Comment:
Out of curiosity: Did prettier add this change? or was it manually included?

Reply:
I don't know.